ansible/cluster.yml

```diff
@@ -65,6 +71,7 @@
 # install kubernetes on the nodes
 - hosts: nodes
   sudo: yes
+  serial: 5
```

Comment:
We need the PR to only contain changes that are required for adding Contiv. Any additional changes that contrib will benefit from, pls create separate pr's.